site2/docs/io-elasticsearch-sink.md

```diff
@@ -6,6 +6,14 @@ sidebar_label: Elasticsearch sink connector
 
 The Elasticsearch sink connector pulls messages from Pulsar topics and persists the messages to indexes.
 
+## Requirements
+
+To deploy an ElasticSearch sink connector, the following are required:
+
+- ElasticSearch 7 (ElasticSearch 8 is not currently supported in Pulsar 2.10)
+- OpenSearch 1.x
+
+The `compatibilityMode` configuration property specifies compatibility with ElasticSearch and OpenSearch clusters.
```

Comment:
Unfortunately this feature is not released yet. Actually this is needed to handle ES 8.